### ISSUE TITLE
RMA not work as expected

### ISSUE THREAD

**Issue by jc-bao** [jc-bao/policy-adaptation-survey#7]
After the adaptation, the training procedure is fairly slow. 

<img width="701" alt="image" src="https://user-images.githubusercontent.com/60093981/209654764-32352d75-1c7e-45c3-817e-6b4d4acb1062.png">

**Comment by jc-bao** [comment]
# Sanity check1: make sure all parameter is observable given the past trajectories.

**Comment by jc-bao** [comment]
# Sanity check2: remove useless prediction parameters and calculate the result again. 

Test case: only keep mass and relative input. Need to learn $m = \frac{F_u}{a+g}$
Result: the loss went down (from 0.15 -> 0.015), but the error is still relatively high.

**Comment by jc-bao** [comment]
# current vector

* How to learn the desired function which maps the trajectory to parameters.

**Comment by jc-bao** [comment]
* update loss function
* SysID paper. 
* Sensitivity analysis.

**Comment by jc-bao** [comment]
# Progress1: with only one parameter m

* The formula is learned with an MSE error of 0.05. 
* However, when adding another parameter in, then the loss goes up to 0.9

**Comment by jc-bao** [comment]
# Trail: compress the input with 3D adaptor

<img width="630" alt="image" src="https://user-images.githubusercontent.com/60093981/210556638-bf952b31-3733-4d82-a905-8d3c825251b6.png">

* Fail: MLP's performance is better than 3D adaptor

**Comment by jc-bao** [comment]
# Trail: permute before reshaping

Problem resolved!

<img width="1468" alt="image" src="https://user-images.githubusercontent.com/60093981/210571342-88a3fce0-4fdf-436b-bd39-28f702914f68.png">

**Comment by jc-bao** [comment]
# Result

<img width="734" alt="image" src="https://user-images.githubusercontent.com/60093981/210676664-27ab2ef4-624b-4994-86fe-3ffbe0b5f4b5.png">
